doc/modules/cross_validation.rst

```diff
@@ -223,10 +222,11 @@ learned using :math:`k - 1` folds, and the fold left out is used for test.
 Example of 2-fold cross-validation on a dataset with 4 samples::
 
   >>> import numpy as np
-  >>> from sklearn.cross_validation import KFold
+  >>> from sklearn.model_selection import KFold
 
-  >>> kf = KFold(4, n_folds=2)
-  >>> for train, test in kf:
+  >>> X = np.ones(4)
```

Comment:
`X = ["a", "b", "c", "d"]` kind of conveys the idea that the data itself aren't used, cause they're strings and not numeric values. Using `object` or `None` would just be confusing. This is not very important, but if you agree it's better, it's a simple enough change.